docs/guides/cloud-secrets.md

```diff
@@ -87,6 +93,15 @@ You can build it via:
 earthly +build
 ```
 
+{% hint style='info' %}
+### Naming of local and cloud-based secrets
+
+The only difference between the naming of locally-supplied and cloud-based secrets is that cloud secrets will contain
+two or more slashes; where as locally-defined secret names will only start with the `+secrets/` suffix,
```

Comment:
Might be worth including an example of a cloud secret key.

Reply:
I called out the fact that cloud secrets start with `+secrets/<user or organization/>`; what do you think about that?

Reply:
sgtm